packages/components/avatar/src/Avatar/Avatar.tsx

```diff
@@ -65,7 +67,9 @@ function _Avatar(
       {...otherProps}
     >
       {isFallback ? (
-        <div className={styles.fallback} data-test-id={`${testId}-fallback`} />
+        <div className={styles.fallback} data-test-id={`${testId}-fallback`}>
+          {initials}
```

Comment:
What happens in a case where there are no initials or image URLs?

Reply:
We will display the same grey background like until now.
You can check it in the deployed version of Storybook [here](https://5fd1dda724cc620021ace8c5-pzscbqwoqw.chromatic.com/?path=/story/components-avatar--overview).